code/analyze/ac/stats/de_StatsFreqPreferences.m

```diff
@@ -51,6 +50,12 @@
           if si == 1
             stats.(prop).mean = cell(n_sigmas, 1);
             stats.(prop).std = cell(n_sigmas, 1);
+            if strcmp(prop, 'freq') % This code is used for sigma vs xover computations
```

Comment:
Nice find. I think pulling everything from line 52 (`stats.(prop).mean = cell(n_sigmas, 1);`) out of the loop makes sense now.

I stuck it in the loop to keep all initialization related to the inner loop together, but since the new storage has nothing to do with it, I think it's useful to go with a more canonical way of initializing (outside the loop for base-level, inside the loop for the rest)